src/gluonnlp/model/utils.py

```diff
@@ -182,6 +185,59 @@ def _get_rnn_cell(mode, num_layers, input_size, hidden_size,
     return rnn_cell
 
 
+def _get_rnn_cell_clip_residual(mode, num_layers, input_size, hidden_size, dropout,
```

Comment:
add clip option to the `_get_rnn_cell` function instead of making a copy.

Reply:
fixed, thanks